### PR TITLE
Add label definitions to configured pipelines

### PR DESCRIPTION
**Pull request by tobim**
You can now add labels to configured pipelines like this:
```yaml
pipelines:
  my-labelled-pipeline:
    name: Labels!
    definition: subscribe foo | publish bar
    labels:
      - critical
      - my-fancy-label
```

These labels do not support custom colors on app.tenzir.com yet.

### CODE REVIEW

**Comment by mavam** [comment]
Mind adding a changelog along with a docs update?

File: contrib/tenzir-plugins
Comment (dominiklohmann) [inline contrib/tenzir-plugins:1]: This unfortunately breaks the app for some reason.

The app also has some naming restrictions for labels: no spaces, max 20 characters.

image

**Review by dominiklohmann — CHANGES_REQUESTED** [review]

**Review by dominiklohmann — APPROVED** [review]

File: changelog/next/features/4247--configured-pipeline-labels.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Pipelines configured as code in the `tenzir.yaml` configuration file may now
+contain labels.
```
Comment (mavam) [inline changelog/next/features/4247--configured-pipeline-labels.md:2]: Please add that to the example in the docs.

Comment (tobim) [reply changelog/next/features/4247--configured-pipeline-labels.md:2]: Already done:
https://github.com/tenzir/tenzir/pull/4247/commits/599d9776abb3a592c8c114921d5332f15adf8812#diff-86b97e9b4d26542ee0333dfd0fe11fc37947daffcb0d87b5d0eecc8948ffa6e9R106-R110

**Pull request merged**